src/quicreach.cpp

```diff
@@ -174,19 +179,21 @@ bool TestReachability(const ReachConfig& Config) {
                 TooMuch = Amplification > 3.0;
                 if (TooMuch) ++TooMuchCount;
             }
-            if (Config.PrintStatistics)
-                printf("    %3u.%03u ms    %3u.%03u ms    %3u.%03u ms    %u:%u %u:%u (%2.1fx)    %4u    %4u    %c",
-                        Connection.Stats.Rtt / 1000, Connection.Stats.Rtt % 1000,
-                        InitialTime / 1000, InitialTime % 1000,
-                        HandshakeTime / 1000, HandshakeTime % 1000,
-                        (uint32_t)Connection.Stats.SendTotalPackets,
-                        (uint32_t)Connection.Stats.RecvTotalPackets,
-                        (uint32_t)Connection.Stats.SendTotalBytes,
-                        (uint32_t)Connection.Stats.RecvTotalBytes,
-                        Amplification,
-                        Connection.Stats.HandshakeClientFlight1Bytes,
-                        Connection.Stats.HandshakeServerFlight1Bytes,
-                        TooMuch ? '!' : (MultiRtt ? '*' : ' '));
+            if (Config.PrintStatistics) {
```

Comment:
Just leave the `{` off

Reply:
We should never have an if statement like that without the brace. The statement contained inside is WAY too large to make it easy to comprehend.

Reply:
In fact I'd prefer to go fix all the other cases in here too. There is no reason to skip braces. Doing so just makes code harder to read.

Reply:
I was trying mostly to follow the Linux style a bit more.

Reply:
Thats a disgusting coding style... But sure whatever

Reply:
I'm not going to argue against you. 😄 You know I like our style, but I was trying to make this more attractive for Linux dev contributors.